media/css/sandstone/buttons.less

```diff
@@ -319,7 +320,6 @@ ul.download-list {
 .linux .download-button .os_linux,
 .linux.x86.x64 .download-button .os_linux64,
 .windows .download-button .os_win,
-.osx .download-button .os_osx,
```

Comment:
Since this line will need to be resurrected when release hits 49, maybe instead of deleting we comment it out with a short explanation?